src/HotChocolate/Spatial/src/Types/Configuration/ISpatialConvention.cs

```diff
@@ -0,0 +1,21 @@
+using HotChocolate.Types.Descriptors;
+using HotChocolate.Types.Spatial.Transformation;
+
+namespace HotChocolate.Types.Spatial.Configuration
+{
+    /// <summary>
+    /// A conventions that configures the behaviour of spatial types
+    /// </summary>
+    public interface ISpatialConvention : IConvention
+    {
+        /// <summary>
+        /// The default SRID/CRS of the schema. All incoming queries will be translated to this SRID
```

Comment:
line length?

Reply:
Living on the edge
![image](https://user-images.githubusercontent.com/14233220/106261714-3c006100-6222-11eb-98f8-f2d06cec6aa6.png)